scripts/secureboot-certs

```diff
@@ -480,10 +480,30 @@ dbx: {}
         metavar="db",
         help="'default' for the default Microsoft certs or a path to a custom auth file.",
     )
+
     install_parser.add_argument(
         "dbx",
         metavar="dbx",
-        help="'none' for no dbx, 'default' for the most recent UEFI dbx, or a path to a custom auth file.",
+        help="""
+'latest' for the most recent UEFI dbx, a path to a custom auth file, or 'none' for no dbx.
+
+Choosing 'none' should be completely avoided in production systems hoping to
+benefit from Secure Boot. It renders Secure Boot practically meaningless
+because attackers may simply load any number of vulnerable binaries that were
+previously signed but later revoked, and thereby take control of the system.
+
+The 'latest' dbx revokes any software that hasn't implemented the most recent
+security fixes, which may include some OS distributions (even if they're
+totally updated, depending how recently the vulnerability was discovered).
+Because it varies per distribution, check if your guest distributions are
+updated to pass the most recent UEFI revocation before installing the latest
+dbx.
+
+For older dbx files, see: https://uefi.org/revocationlistfile/archive. They may passed
```

Comment:
Should "They may passed" be "They may be passed"?

Reply:
Fixed!